firmware/Core/Inc/telecommands/flash_telecommand_defs.h

```diff
@@ -5,6 +5,11 @@
 #include <stdint.h>
 #include "telecommands/telecommand_definitions.h"
 
+/*----------------------------- CONFIG VARIABLES ----------------------------- */
+// NAND Flash Memory Datasheet https://www.farnell.com/datasheets/3151163.pdf (pg.7)
+// Each page is divided into a 2048-byte data storage region, and a 128 bytes spare area (2176 bytes total).
+#define MAX_NUM_BYTES 2048
```

Comment:
Please change to static const uint16_t type. Also, please rename to have FLASH prefix. A better name is FLASH_MAX_BYTES_PER_BLOCK maybe? Or per region? Or something?

Does this constant belong in this file?

Reply:
The only reason I have this constant included in this file is because it is used for telecommand argument conditions and creating global read and write buffers of this size

Reply:
This constant can be removed from this file, thank you for finding this!